book/website/communication/citable/citable-cite.md

```diff
@@ -45,6 +45,10 @@ You can find examples of these statements in the publishers' (research data) aut
 -  “The data that support the findings of this study are available upon request.
 Access conditions and procedures can be found at [URL to restricted access repository such as [EASY](https://easy.dans.knaw.nl/ui/home).]”
 
+**More Data Availability Statement examples:**
```

Comment:
I'm leaning slightly towards saying "Other" in place of "More". But no strong objections to current use.